src/components/Graphs/index.jsx

```diff
@@ -29,12 +29,16 @@ const Graphs = ({ benchmarkData, overviewMode }) => (
     }
     {Object.keys(benchmarkData.graphs)
       .sort(overviewMode ? undefined : sortOverviewFirst)
-      .map(key => benchmarkData.graphs[key])
+      .map((key) => {
+        const graph = benchmarkData.graphs[key];
+        graph.anchor = key;
+        return graph;
```

Comment:
You can do this:
```
return {
   anchorId: key,
   ...benchmarkData.graphs[key],
}
```

Reply:
@armenzg Ok, I'll make a separate element for the anchor. I haven't been able to get to it today, but I will this weekend.

Reply:
@armenzg I added the icon. Let me know if it's okay.

Reply:
Hi @echessa don't forget to address what I mentioend here:
https://github.com/mozilla-frontend-infra/js-perf-dashboard/pull/66/files#r219278602